### PR TITLE
coq: add keybindings for company-coq's documentation functions

### PR DESCRIPTION
**Pull request by siddharthist**
Extra keybindings for the Coq layer concerning documentation

### CODE REVIEW

File: layers/+lang/coq/README.org [hunk 1]
```diff
@@ -70,6 +71,14 @@ font, or disable the feature by adding the following snippet to the your
 | ~SPC m p q~ | Quit prover                                                     |
 | ~SPC m p r~ | Retract buffer - rewinds and moves point to beginning of buffer |
 
+** Getting documentation
+
+| Key Binding | Description                                                     |
+|-------------+-----------------------------------------------------------------|
+| ~SPC m d .~ | Show documentation for whatever is below the cursor             |
```
Comment (sdwolfz) [inline layers/+lang/coq/README.org]: Documentation should be under the `SPC m h` prefix. Also this particular keybinding should be `SPC m h h`.

File: layers/+lang/coq/packages.el [hunk 2]
```diff
@@ -32,12 +32,19 @@
...
+          (car prefix) (cdr prefix)))
       (spacemacs/set-leader-keys-for-major-mode 'coq-mode
         "il" 'company-coq-lemma-from-goal
         "im" 'company-coq-insert-match-construct
-        "ao" 'company-coq-occur))))
+        "ao" 'company-coq-occur
+        "de" 'company-coq-document-error
+        "dE" 'company-coq-browse-error-messages
+        "d." 'company-coq-doc
+        ""))))
```
Comment (sdwolfz) [inline layers/+lang/coq/packages.el]: This empty string should not be here.

**Comment by siddharthist** [comment]
@sdwolfz Thanks for the quick review, I've addressed your comments

**Comment by sdwolfz** [comment]
I can not figure out why the tests are failing here...
@siddharthist can you please rebase with latest develop? maybe that will help. Otherwise I will have to summon the mighty @JAremko for help.

**Comment by JAremko** [comment]
@sdwolfz  CLI interface of documentation tools was updated but since this PR is based on old interface it fails.

**Comment by siddharthist** [comment]
@sdwolfz Rebased!

**Comment by sdwolfz** [comment]
Thanks @siddharthist. I looks like you have a rear formatting problem now:

```diff
diff --git a/layers/+lang/coq/README.org b/layers/+lang/coq/README.org
index ebb52618b..6f06a13af 100644
--- a/layers/+lang/coq/README.org
+++ b/layers/+lang/coq/README.org
@@ -73,11 +73,11 @@ font, or disable the feature by adding the following snippet to the your
 
 ** Getting documentation
 
-| Key Binding | Description                                                     |
-|-------------+-----------------------------------------------------------------|
-| ~SPC m h h~ | Show documentation for whatever is below the cursor             |
-| ~SPC m h e~ | Show documentation for the error in the `*response*` buffer     |
-| ~SPC m h E~ | Browse all available documentation for errors                   |
+| Key Binding | Description                                                 |
+|-------------+-------------------------------------------------------------|
+| ~SPC m h h~ | Show documentation for whatever is below the cursor         |
+| ~SPC m h e~ | Show documentation for the error in the `*response*` buffer |
+| ~SPC m h E~ | Browse all available documentation for errors               |
 
 ** Prover queries
 The mnemonic for =a= is "ask".
```

**Comment by siddharthist** [comment]
@sdwolfz Thanks for the patch! Applied.

**Comment by sdwolfz** [comment]
Thank you :heart:!
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request merged**